eng/pipelines/tagger.yml

```diff
@@ -0,0 +1,90 @@
+        # File containing the package version for Managed/Managed.VS packages.
+        - src/ConfigData/Packages/ProjectSystem/Managed.props
+        # File containing the package version for AppDesigner/Editors packages.
+        - src/ConfigData/Packages/Wizard/packages.props
+
+pool:
+  # Agent Queue: https://devdiv.visualstudio.com/DevDiv/_settings/agentqueues?queueId=3123&view=jobs
+  name: VSEngSS-MicroBuild2022-1ES
+  # Demands Docs: https://docs.microsoft.com/azure/devops/pipelines/process/demands?view=azure-devops&tabs=yaml#manually-entered-demands
+  demands: Cmd
+
+variables:
+  # https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/26284/Enabling-SBOM-For-Your-Component-Insertion-into-VS?anchor=1.-add-the-%27manifest-generator-task%27-to-your-pipeline
+  Packaging.EnableSBOMSigning: true
+  # Opt out of automatically injecting Codesign Validation into the pipeline. We run Codesign Validation as part of the Compliance pipeline.
+  # See: https://aka.ms/gdn-injection
+  runCodesignValidationInjection: false
+  # Suspend enforcement of NuGet Single Feed Policy. See:
+  # - https://aka.ms/nugetmultifeed
+  # - https://docs.opensource.microsoft.com/tools/nuget_security_analysis/nuget_security_analysis/
+  # - https://docs.opensource.microsoft.com/tools/cg/how-to/nuget-multifeed-configuration/
+  # - https://onebranch.visualstudio.com/OneBranch/_wiki/wikis/OneBranch.wiki/5205/TSG-Build-Broken-Due-to-Using-Multiple-Feeds?anchor=setting-nugetsecurityanalysiswarninglevel-in-cdp
+  NugetSecurityAnalysisWarningLevel: none
+
+parameters:
+- name: VsCommitId
+  displayName: VS Commit ID
+  type: string
+  default: $(Build.SourceVersion)
+- name: VsCheckoutDepth
+  displayName: VS Checkout Depth
+  type: number
+  default: 1
+
+###################################################################################################################################################################
+# STEPS
+###################################################################################################################################################################
+
+steps:
+- checkout: self
+  persistCredentials: true
+  # Changes Build.SourcesDirectory from $(Pipeline.Workspace)/s to $(Pipeline.Workspace)/project-system
+  path: project-system
+  # Hard-coded assumption that the commit referenced by the VS insertion is within the last 100 commits in the repo.
+  fetchDepth: 100
+# Create the VS folder and clone the VS repo into it, excluding the files themselves (only .git history is required).
+# Doing this manual checkout over the checkout: VS task saves around 3-4 minutes.
+# The System.AccessToken is required for the clone to occur. See: https://stackoverflow.com/a/56734304/294804
+- powershell: |
+    $null = New-Item -Path '$(Pipeline.Workspace)' -Name VS -ItemType Directory
+    git clone --no-checkout --depth ${{ parameters.VsCheckoutDepth }} https://$(System.AccessToken)@dev.azure.com/devdiv/DevDiv/_git/VS $(Pipeline.Workspace)/VS
```

Comment:
Need to determine better way to check out if depth 1 does not contain the commit that triggered this pipeline (multiple simultaneous insertion PR merges). Can use Build.SourceVersion in some way here.

Reply:
Fixed this by only doing a `git fetch` of the specific commit, which also won't checkout the code. Then, making sure the vsCommitId is used in the script itself, as it is required since there is no branch associated with this folder.